### PR TITLE
fix: use TinaProvider instead of Tina

### PR DESCRIPTION
**Pull request by DirtyF**
Following #877

### CODE REVIEW

File: packages/demo-gatsby/gatsby-browser.js [hunk 1]
```diff
@@ -19,13 +19,11 @@ distributed under the License is distributed on an "AS IS" BASIS,
...
-export function wrapRootElement({element}) {
-    return (
-        <Tina cms={window.tinacms}>
-            {element}
-        </Tina>
-    )
+import * as React from "react"
+import { TinaProvider, TinaCMS } from "tinacms"
+
+const cms = new TinaCMS()
```
Comment (ncphillips) [inline packages/demo-gatsby/gatsby-browser.js]: No need to create that instance
```suggestion
import { TinaProvider } from "tinacms"
```

**Comment by ncphillips** [comment]
@all-contributors please add @DirtyF for code

File: packages/demo-gatsby/gatsby-browser.js (hunk 1 above)
Comment (DirtyF) [reply packages/demo-gatsby/gatsby-browser.js]: I wasn't sure 😊, thanks for the review.

**Comment by dwalkr** [comment]
should this be `chore` and not `fix`?

**Pull request merged**